Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:1.14-alpine
+
+RUN \
+    apk add --no-cache git && \
+    git clone https://github.com/peak/s5cmd && cd s5cmd && \
```

Comment:
Instead of cloning master, I think it would be better to add sources into container then build. This way one can build Docker images from revisions other than master without changing anything. To exclude sources in the final image [multistage build](https://docs.docker.com/develop/develop-images/multistage-build/) could be used.